spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java

```diff
@@ -94,6 +94,10 @@ public String fileScanTaskSetId() {
     return confParser.stringConf().option(SparkReadOptions.FILE_SCAN_TASK_SET_ID).parseOptional();
   }
 
+  public String scanTaskSetId() {
```

Comment:
Shall we also deprecate `fileScanTaskSetId`?

Reply:
Done